generators/app/templates/src/app/screens.ejs

```diff
@@ -38,7 +25,9 @@ class InitialLoadingScreen extends PureComponent {
       nextProps.dispatch(
         NavigationActions.reset({
           index: 0,
-          actions: [NavigationActions.navigate({ routeName: nextProps.currentUser ? 'Home' : 'Login' })]
+          actions: [
+            NavigationActions.navigate({ routeName: nextProps.currentUser ? Routes.Home : Routes.Login })
+          ]
```

Comment:
I think this should be triggered after `initialLoading` goes `false` in an `async action`, so we can get rid of `componentWillReceiveProps` hook. What do you think ?

Reply:
I'd leave it for another pr, since that does not fall within the scope of this card/pr. I'd create a card for this.